core/ui/build.gradle.kts

```diff
@@ -16,6 +16,7 @@ dependencies {
     implementation(project(":navigation"))
     implementation(project(":app-base")) // TODO: temp dependency, remove later
     implementation(project(":math"))
+    implementation("com.github.Vishwa-Raghavendra:colorpicker-compose:v1.1.2")
```

Comment:
Hi @Vishwa-Raghavendra , please add this as a function in `dependencies.kt`. We want to manage all depa from one place.